jest.config.js

```diff
@@ -4,13 +4,13 @@ module.exports = {
   coverageReporters: ['text', 'html'],
   coverageThreshold: {
     global: {
-      branches: 98,
-      functions: 94,
-      lines: 86,
-      statements: 86,
+      branches: 100,
+      functions: 100,
+      lines: 100,
+      statements: 100,
     },
   },
-  moduleFileExtensions: ['ts', 'js', 'json', 'node'],
```

Comment:
I'd recommend we leave this alone. It can result in bizarre test failures if we ever add a dependency that uses the `.node` file extension. I've seen it once on the `@metamask/controllers` repo, which apparently does have such a dependency (or it did once at least).

Reply:
Done in: 6de7a68